.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] [OPTIONAL] you can add this file to your gitignore file. since this is simply a vs code settings file you do not need to track it in your repo.